projects/igniteui-angular/src/lib/grids/common/grid.interface.ts

```diff
@@ -447,7 +440,9 @@ export interface ColumnType extends FieldType {
     rowEnd: number;
     colStart: number;
     colEnd: number;
+    /** @hidden @internal */
     gridRowSpan: number;
+    /** @hidden @internal */
     gridColumnSpan: number;
```

Comment:
these are also present in the cell component... can you please check if there is any necessity for these getters there

Reply:
Only see them on `IgxGridCellComponent` which should not be exposed in any public API, the `CellType` interface doesn't contain those thankfully.